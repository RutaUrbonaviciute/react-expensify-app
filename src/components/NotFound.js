import React from 'react';
import { Link } from 'react-router-dom';

const NotFound = () => (
  <div>
    404 not found bithezz - <Link to="/">go fkin home</Link>
  </div>
)

export default NotFound