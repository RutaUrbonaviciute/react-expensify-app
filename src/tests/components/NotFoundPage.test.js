import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../components/NotFound';

test('should render NotFound page correctly', () => {
  const wrapper = shallow(<NotFound />)
  expect(wrapper).toMatchSnapshot()
})
