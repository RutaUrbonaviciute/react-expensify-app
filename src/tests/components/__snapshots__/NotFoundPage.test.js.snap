// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound page correctly 1`] = `
<div>
  404 not found bithezz - 
  <Link
    to="/"
  >
    go fkin home
  </Link>
</div>
`;
